core/modules/views/plugins/views_plugin_display_page.inc

```diff
@@ -216,7 +216,7 @@ class views_plugin_display_page extends views_plugin_display {
     // display, and arguments should be set on the view.
     $this->view->build();
     if (!empty($this->view->build_info['fail'])) {
-      return backdrop_not_found();
+      return MENU_NOT_FOUND;
```

Comment:
It looks like directly below this, there is a `return backdrop_access_denied();` as well. That should also be replaced with `MENU_ACCESS_DENIED` for the same reasons as this line being changed.

Reply:
fixed.